detect.go

```diff
@@ -105,6 +99,11 @@ func Detect(gemfileParser Parser) packit.DetectFunc {
 		_, err = os.Stat(filepath.Join(context.WorkingDir, "yarn.lock"))
 		if err == nil {
 			requirements = append(requirements, packit.BuildPlanRequirement{
+				Name: "node",
+				Metadata: BuildPlanMetadata{
+					Build: true,
+				},
+			}, packit.BuildPlanRequirement{
```

Comment:
Sorry for commenting on a merged PR, but this change seems to be causing issues with projects still in Rails 6 and/or using sprockets, since those projects do not contain a `yarn.lock` file but still require a JS runtime to compile assets.

Would the team be open to discussing this and either reverting this change, or changing the implementation to accommodate for those cases?

Reply:
Yes. We'd be happy to provide a fix. I think we need some help understanding the specifics of what we should be detecting here. Can you open a new issue and provide an example app?